CMakeLists.txt

```diff
@@ -292,6 +292,10 @@ if (NOT USE_SIMD STREQUAL "")
             else ()
                 add_definitions ("-m${feature}")
             endif ()
+            if (feature STREQUAL "fma" AND CMAKE_COMPILER_IS_CLANG)
+                # Clang needs an explicit flag to make fma ops
+                add_definitions ("-ffp-contract=fast")
```

Comment:
No, fma does not break IEEE, there is no loss of precision. This directive only controls the conversion of a*b+c to fma, it's not like -ffast-math. It's also only necessary on clang, which for some reason (that many would consider a bug) does not automatically use fma even with -mfma is used, whereas gcc does. I expect this will be fixed in some future clang.